Code/Framework/AzToolsFramework/AzToolsFramework/Manipulators/PaintBrushManipulator.h

```diff
@@ -97,18 +67,19 @@ namespace AzToolsFramework
 
         void SetView(AZStd::shared_ptr<ManipulatorViewProjectedCircle> view);
 
-        // Handle mouse events
+        //! Handle mouse events
         bool HandleMouseInteraction(const ViewportInteraction::MouseInteractionEvent& mouseInteraction);
 
-        // PaintBrushRequestBus overrides for getting/setting the paintbrush settings...
-        float GetRadius() const override;
-        float GetIntensity() const override;
-        float GetOpacity() const override;
-        void SetRadius(float radius) override;
-        void SetIntensity(float intensity) override;
-        void SetOpacity(float opacity) override;
+        //! Returns the actions that we want any Component Mode using the Paint Brush Manipulator to support.
+        AZStd::vector<AzToolsFramework::ActionOverride> PopulateActionsImpl();
```

Comment:
update: I believe this is missing an `override` keyword (same as above with `HandleMouseInteraction` and the bus naming `BusName overrides ...`

Reply:
Ah, nope. These are case where the ComponentMode class is the one that inherits from the base class, and this is providing similarly-named methods that get direct-called from the Component Mode class. I guess this might be another argument for creating a base component mode class to disambiguate these a little further. I was following the pattern that I saw in things like EditorWhiteBox*Mode where the ComponentMode calls down to mode->HandleMouseInteraction(), but that pattern isn't strictly required here.